torch/testing/_internal/common_methods_invocations.py

```diff
@@ -539,8 +539,9 @@ def __init__(self,
 
                  # the following are pointers to functions to generate certain classes
                  #   of inputs
-                 sample_inputs_func=None,  # function to generate sample inputs
+                 sample_inputs_func=None,  # function to generate sample inputs with strided-only layouts
                  error_inputs_func=None,  # function to generate inputs that will throw errors
+                 sample_inputs_mixed_func=None,  # function to generate sample inputs with mixed layouts
```

Comment:
I would just add a sample input function for each layout

Reply:
(if we want to enumerate all sample input functions we can have a list of all of them for tests to iterate over)